app/views/surveys/index.html.erb

```diff
@@ -29,7 +29,7 @@
           <td><%= toLocalDateTime(survey.updated_at) %></td>
           <td><%= toLocalDateTime(survey.created_at) %></td>
           <% if editor? %>
-            <td><%= survey.data_provider.name %></td>
+            <td><%= survey.data_provider.try(:name) %></td>
```

Comment:
weshalb ist diese change mit reingerutscht? es gibt 7 dateien mit dem editor code und nur hier jetz ein try. ich werd das hier rausziehen und einen separaten PR machen mit genau der gezielten change in allen dateien...

Reply:
done in 0e89856 hier und #92 für alle anderen index pages